oop_verify.py

```diff
@@ -82,3 +102,211 @@ def spot_check(self):
                     self.failed_pages.append(page)
                     self.pages.pop(page)
         return
+
+
+class ProjectDashboardVerifier(Verifier):
+    def __init__(self):
+        Verifier.__init__(self)
+        self.pages = []
+        self.minimum_size = 410
+        self.page_elements = [
```

Comment:
If page_elements is the same for all the subclasses (or for most subclasses) can it just be made a field of the superclass? And then altered in the subclasses if needed.